templates/zerver/emails/invitation.source.html

```diff
@@ -8,7 +8,7 @@
 <p>{{ _("Hi there,") }}</p>
 
 <p>
-    {% trans %}{{ referrer_full_name }} (<a href="mailto:{{ referrer_email }}">{{ referrer_email }}</a>) wants you to join them on Zulip &mdash; the team communication tool designed for productivity.{% endtrans %}
+    {% trans %}<a href="mailto:{{ referrer_email }}">{{ referrer_full_name }} ({{ referrer_email }}</a>) wants you to join them on Zulip &mdash; the team communication tool designed for productivity.{% endtrans %}
```

Comment:
```suggestion
    {% trans %}<a href="mailto:{{ referrer_email }}">{{ referrer_full_name }} ({{ referrer_email }})</a> wants you to join them on Zulip &mdash; the team communication tool designed for productivity.{% endtrans %}
```

But it would be nice to extract the link as a Jinja macro so the HTML details don’t need to go through translation.